ModerationPolicy.md

```diff
@@ -0,0 +1,355 @@
+# Moderation policy
+
+If you are not a member of the s and wish to submit a
```

Comment:
Typo?

Reply:
Oh, it is. 

Would you mind suggesting the change as well? I could simply accept your suggested change.